### PR TITLE
Test/exercise endpoints

### PR DESCRIPTION
**Pull request by chrisv09**
Added five tests:
- Getting one exercise by name, date and set
- Getting all exercises
- Getting the full exercise history
- Getting all the exercises on a given day

- Getting all the workouts

I also added a package to help us use env files across different OSs.

Remember to npm install after squash + merge!

### CODE REVIEW

**Review by joh748 — CHANGES_REQUESTED** [review]

File: server/controllers/exercises-controller.js [hunk 1]
```diff
@@ -105,9 +132,11 @@ const logExerciseSet = (req, res) => {
         });
 }
 
-module.exports = {
+export {
     exercisesAll,
-    exerciseByNameDateAndSets,
+    exercisesAllHistory,
+    exercisesDay,
+    exerciseByNameDateAndSets
```
[the remark this answers is not in the document]
Comment (chrisv09) [reply server/controllers/exercises-controller.js]: Fixed!

**Review by joh748 — DISMISSED** [review]
otherwise, it looks good to me

**Review by tom1dev — APPROVED** [review]
looks good to me

**Review by LX6T — APPROVED** [review]
very good very good

**Review by joh748 — APPROVED** [review]
looks good to me

**Pull request merged**